features/image-transformations/image-enhancement-and-color-manipulation.md

```diff
@@ -133,3 +133,117 @@ URL - [https://ik.imagekit.io/demo/lipstick_2M.png](https://ik.imagekit.io/demo/
+
+Example usage - `e-gradient-<linear-direction>_<from-color>_<to-color>_<stop-point>`, which would look like, `e-gradient-ld-top_right_from-FF000040_to-00FF0040_sp-0.9`.
+
+{% tabs %}
+{% tab title="Original Image" %}
+URL - [https://ik.imagekit.io/demo/tr:h-300/sample_image.jpg](https://ik.imagekit.io/demo/tr:h-300/sample_image.jpg)
+
+![](https://ik.imagekit.io/demo/tr:h-300/sample_image.jpg)
+{% endtab %}
+
+{% tab title="Default e-gradient" %}
+URL - [https://ik.imagekit.io/demo/tr:h-300,e-gradient/sample_image.jpg](https://ik.imagekit.io/demo/tr:h-300,e-gradient/sample_image.jpg)
+
+![](https://ik.imagekit.io/demo/tr:h-300,e-gradient/sample_image.jpg)
+{% endtab %}
+
+{% tab title="linear-direction=45, from-color=FF000050, to-color=FFFFFF50, sp=0.5" %}
+URL - [https://ik.imagekit.io/demo/tr:h-300,e-gradient-ld-45_from-FF000050_to-FFFFFF50_sp-0.5/sample_image.jpg](https://ik.imagekit.io/demo/tr:h-300,e-gradient-ld-45_from-FF000050_to-FFFFFF50_sp-0.5/sample_image.jpg)
+
+![](https://ik.imagekit.io/demo/tr:h-300,e-gradient-ld-45_from-FF000050_to-FFFFFF50_sp-0.5/sample_image.jpg)
+{% endtab %}
+
+{% tab title="linear-direction=top, from-color=green, to-color=00FF0010, sp=1" %}
+URL - [https://ik.imagekit.io/demo/tr:h-300,e-gradient-ld-top_from-green_to-00FF0010_sp-1/sample_image.jpg](https://ik.imagekit.io/demo/tr:h-300,e-gradient-ld-top_from-green_to-00FF0010_sp-1/sample_image.jpg)
+
+![](https://ik.imagekit.io/demo/tr:h-300,e-gradient-ld-top_from-green_to-00FF0010_sp-1/sample_image.jpg)
+{% endtab %}
+{% endtabs %}
+
+#### Gradient as a background
+
+For input images containing a solid object over a transparent background, gradient can also be applied as an underlay.
+
+##### Usage syntax
+
+```markup
+https://ik.imagekit.io/<imagekitId>/your-image.png?tr=e-gradient-from-red_to-white:l-image,i-your-image.png,t-false,l-end
+```
+
+##### Examples
+
+{% tabs %}
+{% tab title="Original Image" %}
+URL - [https://ik.imagekit.io/demo/tr:w-500/lipstick_2M.png](https://ik.imagekit.io/demo/tr:w-500/lipstick_2M.png)
+
+![](https://ik.imagekit.io/demo/tr:w-500/lipstick_2M.png)
+{% endtab %}
+
+{% tab title="Gradient as a background" %}
+URL - [https://ik.imagekit.io/demo/tr:e-gradient-from-lightskyblue_to-mintcream:l-image,i-lipstick_2M.png,t-false,l-end:w-500/lipstick_2M.png](https://ik.imagekit.io/demo/tr:e-gradient-from-lightskyblue_to-mintcream:l-image,i-lipstick_2M.png,t-false,l-end:w-500/lipstick_2M.png)
+
+![](https://ik.imagekit.io/demo/tr:e-gradient-from-lightskyblue_to-mintcream:l-image,i-lipstick_2M.png,t-false,l-end:w-500/lipstick_2M.png)
+{% endtab %}
+{% endtabs %}
+
+{% hint style="info" %}
+If you experience a positional displacement between the original image and the overlay image, it would be worthwhile to use `t-false` to disable default trimming of the overlay layer like `l-image,i-your-image.png,t-false,l-end`.
+{% endhint %}
+
+#### Adding gradient blocks over images
```

Comment:
We should add a section in the https://docs.imagekit.io/features/image-transformations/overlay-using-layers page too. A short description along with a link to this section can be added there.

Reply:
I confirmed this with @manu4543 if we should add a section or not - it was decided to add a sentence in the solid color section which directs the user to gradient section in image manipulation section.

Reply:
resolved